beacon-chain/state/state-native/state_trie.go

```diff
@@ -731,6 +820,14 @@ func (b *BeaconState) rootSelector(ctx context.Context, field nativetypes.FieldI
 		return stateutil.SyncCommitteeRoot(b.nextSyncCommittee)
 	case nativetypes.LatestExecutionPayloadHeader:
 		return b.latestExecutionPayloadHeader.HashTreeRoot()
+	case nativetypes.LatestExecutionPayloadHeaderCapella:
+		return b.latestExecutionPayloadHeaderCapella.HashTreeRoot()
+	case nativetypes.WithdrawalQueue:
+		return [32]byte{}, errors.New("not implemented")
```

Comment:
and one more @potuz

Reply:
Same as above, why doesn't fastssz already cover this give it's a list?

Reply:
its not hard to do custom hashing for this too.